odpf/entropy/v1beta1/resource.proto

```diff
@@ -93,6 +101,7 @@ message ResourceState {
   Status status = 1;
   google.protobuf.Value output = 2;
   bytes module_data = 3;
+  LogOptions log = 4;
```

Comment:
what do you think about the name `log_options` for the field?

Reply:
done